site/docs/community.md

```diff
@@ -100,3 +102,56 @@ Recruitment of community members should not be conducted through direct messages
 related to contributing to or using Iceberg can be posted to the `#jobs` channel. 
 
 For questions regarding any of the guidelines above, please contact a PMC member
+
+## The Path from Contributor to Committer
+
+Many contributors have questions about how to become a committer. This section outlines what committers do and how they are invited.
+
+### What are the responsibilities of a committer?
+
+In the Iceberg project, committers are community members that can review and commit changes to Iceberg repositories. Reviewing is the primary responsibility of committers.
+
+### How are new committers added?
+
+Starting from the foundation guidelines, committers are nominated and discussed by the PMC, which uses a consensus vote to confirm a new committer. This vote is the only formal requirement in the Iceberg community — there are no other requirements, such as a minimum period of time or a minimum number of contributions. Similarly, there is no length of time or number of commits that automatically qualify someone to be a committer.
+
+Committers are added when someone has built trust with PMC members that they have good judgment and are a reliable reviewer.
```

Comment:
That's OK to say it's one of the main criteria (it's up to the PMC). I would just mention https://community.apache.org/pmc/adding-committers.html as comdev good practive.

Reply:
Hey JB, thanks for linking that page. I was surprised that this page changed so substantially to how it was before, as an effort of deduplication: https://github.com/apache/comdev-site/pull/191/files#diff-f61968335feadda8b65024fba3abbee7ffe36f56ad106968f47927a9b286f290

This is my take. It depends on the project. For another project I worked on, the risk of accidentally committing a bug is fairly low, while on Iceberg there is a chance of data loss or incorrectness. Which would be a real threat to the project. As always, wording is subjective, and also cultural. If we feel like extending this, I would rather go with something like: 

> Someone who's committed to the project and helping the community.

But I would leave it as it is now as it boils down to the essence, and it isn't set in stone either (in git actually).

Reply:
Thanks for the feedback, JB. Right now, we're focusing on describing how the community operates, rather than discussing how it might operate in the future.

Reply:
Yes, sure, each PMC is defining the criteria, and expectations. As soon as it's consistent across the project, that's totally fine (I think it's important that the criteria is the same on iceberg, iceberg-rust, iceberg-python, ... else it's not fair for contributors).

@Fokko I don't see why Iceberg would be different compared to other mission critical projects we have at Apache (like ActiveMQ, Kafka, ...). Your proposal is fine to me and it's what I meant. It's basically a statement that we value non-code contributions.

I just linked the page as information (not as "strong guideline" or template to use).